includes/Classifai/Providers/Azure/ComputerVision.php

```diff
@@ -770,8 +827,8 @@ public function setup_fields_sections() {
 		);
 		add_settings_field(
 			'enable-image-captions',
-			esc_html__( 'Generate alt text', 'classifai' ),
-			[ $this, 'render_input' ],
+			esc_html__( 'Generate descriptive text', 'classifai' ),
```

Comment:
`Alt text confidence threshold` will need to become `Descriptive text confidence threshold` in the next option.